lib/assets/javascripts/twine.js.coffee

```diff
@@ -274,7 +280,11 @@ Twine.bindingTypes =
     return refresh: ->
       newValue = fn.call(node, context, rootContext)
       for key, value of newValue when lastValue[key] != value
-        $(node).attr(key, value || null)
+        if value
+          value = value.call() if typeof value == 'function'
```

Comment:
I'm uncertain that there could be an occurence of `value` being a function that needs arguments.  I guess if you did `bind-placeholder="someFunction(foo, bar, baz)"` and `someFunction()` itself returned yet another function, then `value` could be a function for `$.attr` to use.  Am I correct in understanding this is the scenario at hand?

In the jQuery impl, I don't think it could ever call that fn with any arguments, since the destructuring into `key, value` there would yield 1 key and 1 single object for `value`.

I think - tentatively - you're OK with not allowing arguments since there should be no way to pass them through in the jQ implementation.